granary/microformats2.py

```diff
@@ -293,6 +295,17 @@ def json_to_object(mf2):
     'tags': [json_to_object(cat) for cat in props.get('category', [])],
   }
 
+  lat, lng = prop.get('latitude'), prop.get('longitude')
+  if lat and lng:
+    try:
+      lat, lng = float(lat), float(lng)
+      # 2 leading places for latitude, 3 for longitude
+      obj['position'] = '%0+10.6f%0+11.6f/' % (lat, lng)
```

Comment:
oh actually, I think this is only called when converting silos to AS. Should I add a call to `postprocess_activity` in `microformats2.html_to_activities`?